### PR TITLE
SC-10203: Initial docs for private locations

### PR DESCRIPTION
**Pull request by sivasamyk**

### CODE REVIEW

File: docs/synthetics/browser-monitor.md [hunk 1]
```diff
@@ -39,6 +39,12 @@ By default, the UI adds the below conditions while creating a browser monitor. Y
 
 The browser monitor script allows the collection of page screenshots at any point during the execution. This can be collected using [page.screenshot()](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagescreenshotoptions) Puppeteer API. JPEG and PNG image types are supported. Currently, the number of screenshots per run is limited to one. On failure of the script due to errors like navigation timeout, assertion failed, etc., a screenshot `error.png` will be collected for analysis.
 
+## Console Errors
+
+The browser logs any errors while loading the web pages such as javascript errors, network errors, etc in the browser console. The browser monitor collects these errors. These errors can be used to debug run failures. These errors are shown in the `Logs` section of run details.
```
Comment (otisg) [inline docs/synthetics/browser-monitor.md:44]: etc => etc.
these errors. These errors => such errors. These errors
failures. These errors => failures and they

File: docs/synthetics/private-locations.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+title: Private Locations
+description: Private locations allows you to monitor the APIs and webpages that are not accessible from the Internet such as your internal websites, private APIs, etc
```
Comment (otisg) [inline docs/synthetics/private-locations.md:2]: Private locations allows => Private locations functionality allows
etc => etc.

File: docs/synthetics/private-locations.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Private locations allow you to run the synthetics tests from your private network. Some of the use cases for using private locations are:
```
Comment (otisg) [inline docs/synthetics/private-locations.md:4]: synthetics tests => synthetic tests

File: docs/synthetics/private-locations.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Create Private Location
+
+To create a private location, navigate to Synthetics -> Private Locations -> New Private Location. 
+
+![Create Private Location]()
+
+* **Name** - Unique name for the private location
+* **Description** - A brief description of the location
+* **City** - Select the city where the machine running the agent is physically located. This is used to calculate latitude/longitude to plot the location in the map. If your city is not listed, select the nearest city.
```
Comment (otisg) [inline docs/synthetics/private-locations.md:18]: This is used to calculate latitude/longitude to plot the location in the map. If your city is not listed, select the nearest city. => If your city is not listed, select the nearest city. This is used to calculate latitude/longitude to plot the location in the map. 
(just swapping sentences, a little easier for reader to follow)

File: docs/synthetics/private-locations.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Each private location is assigned a unique token. You can disable or delete a location, when not in use.
+
+## Install Private Agents
+
+After creating the private location from UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images one for each HTTP & Browser monitor respectively.
```
Comment (otisg) [inline docs/synthetics/private-locations.md:24]: There are two Docker images one for each HTTP & Browser monitor respectively.
=>
There are two Docker images, one for HTTP monitor and another for Browser monitor.

File: docs/synthetics/private-locations.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+* **Name** - Unique name for the private location
+* **Description** - A brief description of the location
+* **City** - Select the city where the machine running the agent is physically located. This is used to calculate latitude/longitude to plot the location in the map. If your city is not listed, select the nearest city.
+
+Each private location is assigned a unique token. You can disable or delete a location, when not in use.
+
+## Install Private Agents
+
+After creating the private location from UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images one for each HTTP & Browser monitor respectively. 
+You can deploy both agents or one based on your monitor types. Follow the instructions in the UI to install private agents in your network.
```
Comment (otisg) [inline docs/synthetics/private-locations.md:25]: > You can deploy both agents or one based on your monitor types

Not sure what you mean by "on your monitor types"?

Maybe this should be:
You can deploy either the HTTP or Browser monitor or both.
?

**Review by otisg — APPROVED** [review]
LGTM, just monitor mods needed.

File: docs/synthetics/browser-monitor.md [hunk 7]
```diff
@@ -39,6 +39,12 @@ By default, the UI adds the below conditions while creating a browser monitor. Y
 
 The browser monitor script allows the collection of page screenshots at any point during the execution. This can be collected using [page.screenshot()](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagescreenshotoptions) Puppeteer API. JPEG and PNG image types are supported. Currently, the number of screenshots per run is limited to one. On failure of the script due to errors like navigation timeout, assertion failed, etc., a screenshot `error.png` will be collected for analysis.
 
+## Console Errors
+
+The browser logs any errors while loading the web pages such as javascript errors, network errors, etc. in the browser console. The browser monitor collects such errors. These errors can be used to debug run failures and they are shown in the `Logs` section of run details.
```
Comment (adnanrahic) [inline docs/synthetics/browser-monitor.md:44]:
```suggestion
The browser logs any errors while loading the web pages such as JavaScript errors, network errors, etc. in the browser console. The browser monitor collects such errors. These errors can be used to debug run failures and they are shown in the `Logs` section of run details.
```

File: docs/synthetics/private-locations.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+title: Private Locations
+description: Private locations functionality allows you to monitor the APIs and webpages that are not accessible from the Internet such as your internal websites, private APIs, etc.
```
Comment (adnanrahic) [inline docs/synthetics/private-locations.md:2]:
```suggestion
description: The private locations feature allows you to monitor APIs and webpages that are not accessible from the Internet such as your internal websites, private APIs, etc.
```

File: docs/synthetics/private-locations.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+Private locations functionality allows you to run the synthetic tests from your private network. Some of the use cases for using private locations are:
```
Comment (adnanrahic) [inline docs/synthetics/private-locations.md:4]:
```suggestion
The private locations feature allows you to run synthetic tests from your private network. Some of the use cases for using private locations are:
```

File: docs/synthetics/private-locations.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+* **Name** - Unique name for the private location
+* **Description** - A brief description of the location
+* **City** - Select the city where the machine running the agent is physically located. If your city is not listed, select the nearest city. This is used to calculate latitude/longitude to plot the location in the map.
+
+Each private location is assigned a unique token. You can disable or delete a location, when not in use.
+
+## Install Private Agents
+
+After creating the private location from UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images, one for HTTP monitor and another for Browser monitor.
```
Comment (adnanrahic) [inline docs/synthetics/private-locations.md:24]:
```suggestion
After creating the private location from the UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images, one for HTTP monitors and another for Browser monitors.
```

File: docs/synthetics/private-locations.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+* Monitor internal APIs or web applications that are not accessible from the Internet
+* Compare the performance of your applications between the Internet & your private network
+* Monitor your applications from specific locations that are not supported by Sematext
+
+## Create Private Location
+
+To create a private location, navigate to Synthetics -> Private Locations -> New Private Location. 
+
+![Create Private Location]()
```
Comment (adnanrahic) [inline docs/synthetics/private-locations.md:14]: No link?

File: docs/synthetics/private-locations.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+* **City** - Select the city where the machine running the agent is physically located. If your city is not listed, select the nearest city. This is used to calculate latitude/longitude to plot the location in the map.
+
+Each private location is assigned a unique token. You can disable or delete a location, when not in use.
+
+## Install Private Agents
+
+After creating the private location from UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images, one for HTTP monitor and another for Browser monitor.
+You can deploy either the HTTP or Browser monitor or both. Follow the instructions in the UI to install private agents in your network.
+
+![Agent Installation Instructions]()
```
Comment (adnanrahic) [inline docs/synthetics/private-locations.md:27]: No link?

File: docs/synthetics/private-locations.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+If you have multiple private locations, you need to install agents individually in each location. Private agents use the HTTPS protocol to periodically fetch the monitors to run and send the result to Sematext Cloud. If you have a firewall configured in your network, you need to open a bidirectional HTTPS traffic for `synthetics-scheduler.sematext.com` and `synthetics-receiver.sematext.com` hosts. For the EU region, the hosts are `synthetics-scheduler.eu.sematext.com` and `synthetics-receiver.eu.sematext.com`.
```
Comment (adnanrahic) [inline docs/synthetics/private-locations.md:29]:
```suggestion
If you have multiple private locations, you need to install agents individually in each location. Private agents use the HTTPS protocol to periodically fetch the monitors to run and send the result to Sematext Cloud. If you have a firewall configured in your network, you need to open bidirectional HTTPS traffic for the `synthetics-scheduler.sematext.com` and `synthetics-receiver.sematext.com` hosts. For the EU region, the hosts are `synthetics-scheduler.eu.sematext.com` and `synthetics-receiver.eu.sematext.com`.
```

File: docs/synthetics/private-locations.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+## Using Private Locations
+
+Once you create a private location from UI and install the agents, you can use it while creating the monitors or update existing monitors to run from the private location. Private location behaves exactly like public locations.
```
Comment (adnanrahic) [inline docs/synthetics/private-locations.md:33]:
```suggestion
Once you create a private location from the UI and install the agents, you can use it while creating the monitors or update existing monitors to run from the private location. Private locations behave exactly the same as public locations. 
```

File: docs/synthetics/private-locations.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+
+If you have multiple private locations, you need to install agents individually in each location. Private agents use the HTTPS protocol to periodically fetch the monitors to run and send the result to Sematext Cloud. If you have a firewall configured in your network, you need to open a bidirectional HTTPS traffic for `synthetics-scheduler.sematext.com` and `synthetics-receiver.sematext.com` hosts. For the EU region, the hosts are `synthetics-scheduler.eu.sematext.com` and `synthetics-receiver.eu.sematext.com`.
+
+## Using Private Locations
+
+Once you create a private location from UI and install the agents, you can use it while creating the monitors or update existing monitors to run from the private location. Private location behaves exactly like public locations. 
+
+![Create Monitor with Private Location](../images/synthetics/private-locations-monitor.png)
+
+![Monitor Overview with Private Location]()
```
Comment (adnanrahic) [inline docs/synthetics/private-locations.md:37]: No link?

**Review by adnanrahic — CHANGES_REQUESTED** [review]
Just a few edits and some missing images.

File: docs/synthetics/private-locations.md (hunk 6 above)
Comment (sivasamyk) [reply docs/synthetics/private-locations.md]: Yes. I will change to "You can deploy either the HTTP or Browser monitor or both."

**Pull request merged**